src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -996,7 +996,7 @@ class CodeBlockPart extends Disposable implements IChatResultCodeBlockPart {
 	}
 
 	private setText(newText: string): void {
-		const currentText = this.textModel.getLinesContent().join('\n');
```

Comment:
It would probably be possible to do this diff faster, if we're worried about speed, but these should be small